src/main/java/frc/robot/Robot.java

```diff
@@ -83,7 +88,16 @@ public void teleopInit() {
 
   /** This function is called periodically during operator control. */
   @Override
-  public void teleopPeriodic() {}
+  public void teleopPeriodic() {
+   // m_robotContainer.s_Pivot.setRotationVoltage(m_robotContainer.s_Pivot.calculateRotationVoltage(90));
```

Comment:
Understandably, we have rapid development, but I would avoid leaving commented out code without a comment describing the why/what this code is doing (seen a few times, just left a comment here)